docs/content/getting-started/setup-openfga.mdx

```diff
@@ -188,6 +188,20 @@ If you need to serve the profiler on a different address than the default `:3001
 docker run -p 8080:8080 -p 8081:8081 -p 3000:3000 -p 3002:3002 openfga/openfga run --profiler-enabled --profiler-addr :3002
 ```
 
+## Health Check
+
+The OpenFGA is configured with a HTTP health check endpoint `/healthz` that is wired to datastore testing. Possible values are
```

Comment:
```suggestion
OpenFGA is configured with a HTTP health check endpoint `/healthz` that is wired to datastore testing. Possible values are
```

Reply:
It also has a gRPC health check I believe. Should we mention that too?

Reply:
> Should we mention that too?

yes

Reply:
Added